### ISSUE TITLE
Submit to rOpenSci?

### ISSUE THREAD

**Issue by Aariq** [cct-datascience/azmetr#10]
Not sure if it's in scope or not because of the narrow geographical range, but it might be worthwhile putting in a [pre-submission inquiry](https://devguide.ropensci.org/authors-guide.html)

**Comment by jeremylweiss** [comment]
Good idea, let’s please keep this in mind as we work on things this fall


From: Eric R. Scott ***@***.***>
Date: Monday, October 10, 2022 at 7:49 AM
To: cct-datascience/azmetr ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [EXT][cct-datascience/azmetr] Submit to rOpenSci? (Issue #10)

External Email

Not sure if it's in scope or not because of the narrow geographical range, but it might be worthwhile putting in a pre-submission inquiry<https://devguide.ropensci.org/authors-guide.html>
